### ISSUE TITLE
FAILED | Difference exceeds tolerance (rel=1e-05, abs=1e-05)

### ISSUE THREAD

**Issue by Lee-ray-a** [NVIDIA/TensorRT#2415]
## Description

**Backgroud**: I was quantify yolov7.onnx to trt int8, and the mAP was reduced by 10%, so I used polygraph to find out which layer went wrong during the optimization.
**This is my command**: ``run best.onnx --trt --int8 --onnxrt``
**This is my result:**
[compare.txt](https://github.com/NVIDIA/TensorRT/files/9858304/compare.txt)

## Environment

**TensorRT Version**: 
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**:

**Comment by zerollzeng** [comment]
> and the mAP was reduced by 10%,

This is due to training

About the polygraphy results, there might be a bug in TRT or it may be caused by numerical underflow/overflow. can you share your onnx model with us?

**Comment by Lee-ray-a** [comment]
> 
**This is my onnx file, and it has two labels, [helmet, head]**
https://drive.google.com/file/d/1m2F9Z-Qn-mFJyVM1tnRztJbVYV3UuhhL/view?usp=sharing

Thanks for your reply

**Comment by zerollzeng** [comment]
@pranavm-nvidia when running `polygraphy run best.onnx --trt --int8 --onnxrt` with a QAT model(maybe just some layers are running in INT8 explicitly), will TRT do the fake PTQ on the network? like do PTQ calibration or just specify dynamic ranges for the FP32 layers.

**Comment by pranavm-nvidia** [comment]
I'm not actually sure. @ttyio do you know?

**Comment by ttyio** [comment]
> @pranavm-nvidia when running `polygraphy run best.onnx --trt --int8 --onnxrt` with a QAT model(maybe just some layers are running in INT8 explicitly), will TRT do the fake PTQ on the network? like do PTQ calibration or just specify dynamic ranges for the FP32 layers.

No, QAT fine tuning is done in pytorch quantization toolkit, once exported as onnx with Q/DQ nodes, we will no longer fine tune it, thanks!

**Comment by zerollzeng** [comment]
> **This is my onnx file, and it has two labels, [helmet, head]** https://drive.google.com/file/d/1m2F9Z-Qn-mFJyVM1tnRztJbVYV3UuhhL/view?usp=sharing
> 
> Thanks for your reply

Can you also provide a real input to this networks? e.g. with polygraphy data_loader.py

**Comment by ttyio** [comment]
closing due to no activity for more than 3 weeks, please reopen if you still have question, thanks!
